### ISSUE TITLE
Just Black window

### ISSUE THREAD

**Issue by keithy** [vercel/hyper#4635]
As a Hyper virgin, I downloaded the latest from the website, installed on Mac OS High Sierra, and the result... Just an all black window.

- Mac OS X High Sierra
- 3.0..2
- no changes made

**Comment by LopezPedrosaAlex** [comment]
Same issue here

**Comment by LabhanshAgrawal** [comment]
try the latest build https://github.com/vercel/hyper/actions/runs/137478211

**Comment by LopezPedrosaAlex** [comment]
It still doesn't work for me.

When I open Hyper it pops up like that:


![Captura de pantalla 2020-06-17 a las 10 29 08](https://user-images.githubusercontent.com/58748526/84874800-692e4380-b085-11ea-935c-c4f267182404.png)

**Comment by LabhanshAgrawal** [comment]
can you check the developer tools once, and see if there are any errors.

**Comment by LopezPedrosaAlex** [comment]
Yes! Something about WebGL 2.0 is not supported in my machine. I have already tried of disabling "webGLRenderer" to false in the file ".hyper.js". But not working still.

**Comment by LabhanshAgrawal** [comment]
current canary has webgl disabled, can you post a screenshot of the devtools

**Comment by LopezPedrosaAlex** [comment]
Not even looking good with canary.

![Captura de pantalla 2020-06-17 a las 15 42 23](https://user-images.githubusercontent.com/58748526/84905634-326e2280-b0b1-11ea-919e-daaee16ec879.png)

**Comment by LabhanshAgrawal** [comment]
Does it look like that? with all the weird artifacts or did the image got corrupted while uploading?
can you reupload with the console tab opened instead of elements?

**Comment by LopezPedrosaAlex** [comment]
In canary it looks like that, I don't know why. In Hyper 3.0.2 doesn't look like that so I downloaded again this version to show you the console tab.

![Captura de pantalla 2020-06-18 a las 12 50 56](https://user-images.githubusercontent.com/58748526/85011801-5dfd1580-b162-11ea-82d9-d71924a63d0e.png)

**Comment by LabhanshAgrawal** [comment]
Can you try the steps in https://github.com/vercel/hyper#contribute and see if it works for local build

**Comment by LopezPedrosaAlex** [comment]
I have tried and it's still the same. Although there is a step I couldn't do with "yarn" (yarn run dev), cuz it pops up this:

![Captura de pantalla 2020-06-18 a las 17 01 47](https://user-images.githubusercontent.com/58748526/85037571-d6c19900-b185-11ea-9224-f2f2875cdb80.png)

**Comment by LabhanshAgrawal** [comment]
did you run `yarn` first? to install all the deps.

**Comment by LopezPedrosaAlex** [comment]
Oh, I'm so dumb. That was it. Now I can run this command until a part where I have to stop the process because it's not doing anything. And Hyper is not working still.

![Captura de pantalla 2020-06-19 a las 13 47 05](https://user-images.githubusercontent.com/58748526/85129596-a801fc00-b233-11ea-8e89-e7f4fa9ae1d7.png)

**Comment by LabhanshAgrawal** [comment]
> Oh, I'm so dumb. That was it. Now I can run this command until a part where I have to stop the process because it's not doing anything. And Hyper is not working still.
> 
> ![Captura de pantalla 2020-06-19 a las 13 47 05](https://user-images.githubusercontent.com/58748526/85129596-a801fc00-b233-11ea-8e89-e7f4fa9ae1d7.png)

you have to open two terminals, run `yarn run dev` in one and once it gets to the above state, run `yarn run app` in the other.

**Comment by LopezPedrosaAlex** [comment]
Black window still... 

![Captura de pantalla 2020-06-19 a las 14 23 01](https://user-images.githubusercontent.com/58748526/85132263-b1da2e00-b238-11ea-900c-31dc4a9fbb56.png)

**Comment by LabhanshAgrawal** [comment]
Are you using High Sierra?

**Comment by LopezPedrosaAlex** [comment]
Yes! High Sierra 10.13.6

**Comment by LabhanshAgrawal** [comment]
I found an issue https://github.com/electron/electron/issues/10736 with electron and high sierra. Maybe you are hitting this. In it someone mentioned facing issues with canvas rendering, so try out the build from https://github.com/labhanshagrawal/hyper/actions/runs/139451227 it has webgl renderer enabled.
